src/library-authoring/LibraryBlock/LibraryBlock.tsx

```diff
@@ -36,15 +37,21 @@ export const LibraryBlock = ({
   minHeight,
   scrolling = 'no',
   scrollIntoView = false,
+  isBlockV1 = false,
 }: LibraryBlockProps) => {
   const { iframeRef, setIframeRef } = useIframe();
   const xblockView = view ?? 'student_view';
 
   const studioBaseUrl = getConfig().STUDIO_BASE_URL;
+  const lmsBaseUrl = getConfig().LMS_BASE_URL;
 
   const intl = useIntl();
   const queryStr = version ? `?version=${version}` : '';
-  const iframeUrl = `${studioBaseUrl}/xblocks/v2/${usageKey}/embed/${xblockView}/${queryStr}`;
+
+  // For now, always show the draft version of the Xblock v1
```

Comment:
```suggestion
  // For now, always show the draft version of the Xblock v1
  // It would be better to use a Studio URL for this, but there is no embeddable URL
  // to render a single component in the Studio API as far as we can tell.
```

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/2449/commits/c8a3dcdfd37eeb4a0aec086cf7a23bb58a6a9b64